smart-contracts/contracts/mixins/CloneFactory.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.5.11;
+
+// From https://github.com/optionality/clone-factory/blob/master/contracts/CloneFactory.sol
+// Updated to support Solidity 5
+
+contract CloneFactory {
+  function createClone(address target) internal returns (address result) {
```

Comment:
I see there's a `isClone` function available if we need it as well.

Reply:
Yup.  I left it out though to save gas.  Knowing it's a clone doesn't help us much, we still need to confirm that Unlock.sol created it (or check the code itself).

Reply:
copy that.